MediaAdminBundle/Tests/Functional/EventSubscriber/UpdateMediaReferenceSubscriberTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace OpenOrchestra\MediaAdminBundle\Tests\Functional\EventSubscriber;
+
+use OpenOrchestra\Media\Model\MediaInterface;
+use OpenOrchestra\MediaModelBundle\Document\Media;
+use OpenOrchestra\ModelBundle\Document\Block;
+use OpenOrchestra\ModelBundle\Document\Node;
+use OpenOrchestra\BackofficeBundle\Tests\Functional\Controller\AbstractControllerTest;
+use OpenOrchestra\ModelBundle\Document\Status;
+use OpenOrchestra\ModelInterface\Event\StatusableEvent;
+use OpenOrchestra\ModelInterface\StatusEvents;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+class UpdateMediaReferenceSubscriberTest extends AbstractControllerTest
```

Comment:
@nicolasThal  : Client need to be authenticated to change status when it dispatch an event.